src/ert/_c_wrappers/enkf/lark_parser.py

```diff
@@ -68,59 +200,60 @@ def __default_token__(self, token) -> FileContextToken:
         return FileContextToken(token, self.filename)
 
 
-grammar = r"""
-WHITESPACE: (" "|"\t")+
-%ignore WHITESPACE
+class InstructionTransformer(Transformer):
+    def job_instruction(self, node):
```

Comment:
(also applies for `regular_instruction`) Not seeing this used, assume it is used by some lark magic in the background, maybe just a short comment on how/why this is needed, as it is not used "directly" in code but only appears in the grammar itself, but maybe this is also just me not understanding exactly how lark works.